.gitignore

```diff
@@ -0,0 +1 @@
+.test.md
```

Comment:
Please ensure that this file is added to `gitignore file` according to the requirement.